test/unit/componentHandler.js

```diff
@@ -76,9 +76,9 @@ describe('componentHandler', function() {
 
   it('should upgrade a single component to an element by using its CSS classes', function() {
     var el = document.createElement('button');
-    el.className = 'mdl-button mdl-js-button';
+    el.className = 'mdl-js-button mdl-js-ripple-effect';
     componentHandler.upgradeElement(el);
-    expect($(el)).to.have.data('upgraded', ',MaterialButton');
+    expect($(el)).to.have.data('upgraded', ',MaterialButton,MaterialRipple');
```

Comment:
That's odd. Always worked for me... Maybe I just wasn't checking the underside of things, just the UX.